<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLSLiveInfoNaverTV</class>
 <widget class="QWidget" name="PLSLiveInfoNaverTV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveInfo.liveinformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="nvtTitleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>195</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>195</height>
      </size>
     </property>
     <widget class="QLabel" name="nvtTitleLabel">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>NAVER TV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="PLSSelectImageButton" name="thumbnailButton">
      <property name="geometry">
       <rect>
        <x>253</x>
        <y>35</y>
        <width>214</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>214</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>214</width>
        <height>120</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nvtGuidelabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>581</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="midwidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="scheduleLabel">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>LiveInfo.Youtube.scheduled.title</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="PLSScheduleCombox" name="scheCombox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="liveTitleLabel">
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>LiveInfo.base.Title</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditTitle">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="bottomFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
        <property name="useFor" stdset="0">
         <string notr="true">Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="useFor" stdset="0">
         <string notr="true">Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rehearsalButton">
        <property name="text">
         <string>Rehearsal</string>
        </property>
        <property name="useFor" stdset="0">
         <string notr="true">Rehearsal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PLSScheduleCombox</class>
   <extends>QPushButton</extends>
   <header>PLSScheduleCombox.h</header>
  </customwidget>
  <customwidget>
   <class>PLSSelectImageButton</class>
   <extends>QLabel</extends>
   <header>plsselectimagebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
